web_accounts_list.json

```
@@ -1374,6 +1374,17 @@
          "category" : "gaming",
          "valid" : true
       },
+      {
+         "name" : "Quora",
+         "check_uri" : "https://{account}.quora.com/profile/",
+         "account_existence_code" : "200",
+         "account_existence_string" : " - Quora</title>",
+         "account_missing_string" : "",
+         "account_missing_code" : "302",

```

```suggestion
         "account_missing_code" : "200",
```

The implementation of the Python `requests` library follows redirects. As such, in this instance, the HTTP status code used for matching is `200`, rather than `302`.